scipy/stats/_survival.py

```diff
@@ -410,29 +410,30 @@ def logrank(
     Parameters
     ----------
     x, y : array_like or CensoredData
-        Samples to compare based on their survival functions.
+        Samples to compare based on their empirical survival functions.
     alternative : {'two-sided', 'less', 'greater'}, optional
         Defines the alternative hypothesis.
 
-        The null hypothesis is that the survival functions underlying
-        `x` and `y` are equal. The following alternative
+        The null hypothesis is that the survival functions of the distributions
+        underlying `x` and `y` are identical. The following alternative
```

Comment:
Technically, we should probably always write "survival functions _of the distributions_ underlying..." because survival functions are properties of distributions, and distributions are what "underly" the samples. But that is very verbose, so I think it's OK to just add it here.